be/src/agent/task_worker_pool.cpp

```diff
@@ -1683,9 +1701,64 @@ void TaskWorkerPool::_submit_table_compaction_worker_thread_callback() {
     }
 }
 
-void TaskWorkerPool::_storage_medium_migrate_v2_worker_thread_callback() {
+void TaskWorkerPool::_storage_refresh_storage_policy_worker_thread_callback() {
```

Comment:
why do you delete _storage_medium_migrate_v2_worker_thread_callback？do you want to implement it in BE?